### DISCUSSION TITLE
Security Onion Setup: 2.3.110

### DISCUSSION THREAD

**Discussion by dhill2022** [Security-Onion-Solutions/securityonion#7690] [Unsupported Versions]
![Sec_onion_install_error](https://user-images.githubusercontent.com/102825169/161315510-d9d1df9b-8ed7-4591-b040-c70727a9d3df.PNG)

Getting error on install. Using this as a Syslog server to pull in logs from multiple sources (20 or more).

**Comment by tomjones1977** [comment]
What does the sosetup.log file say?

**Comment by dougburks** [comment]
@dhill2022 Not sure if you included your sosetup.log, but it's not showing up. Please try attaching it via the web interface.
